package.json

```diff
@@ -7,17 +7,14 @@
   "scripts": {
     "lerna:version": "lerna version",
     "lerna:publish": "lerna publish from-git --yes",
-    "ci": "npm-run-all download build lint format:validate test clean",
+    "ci": "npm-run-all build lint format:validate test",
```

Comment:
Why do we avoid the downloads?

Reply:
I think downloading 2mb (supposed we don't download the license with it) each time we want to run the parser makes us lose a lot of time. Also, according to the [license](https://www.unicode.org/license.html), by downloading the file the user still accepts the license.
Also, as another solution we could just use another node_module to do the same work as we do. (i.e [npm unicode](https://github.com/eversport/node-unicodetable))
That is why I would either store it locally in the repo or use something else rather than downloading the file each time. Plus, according to the [Java Specs](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Character.html#isDefined(int)), it seems Java themselves use the UnicodeData.txt file

Reply:
It is not each time we use the parser but each time we run the CI process.
And you could even do it so it would not run as part of the normal CI process 
but be a manual step that would be run manually only if there were changes to the generated .js file for the unicode data.

Just make sure that we comply with all the legal requirements if we include this as part of the repository and also filter it out of the npm package we are releasing (we do not need it at runtime).

Reply:
I think everything is fine as long as we issue a copy of the license and copyrights in the same folder as the UnicodeData.txt file (just as I did it). I'm thinking of changing the .html file to a .md file just so that we can remove the .css file to make it less intrusive in the project.

If we have the UnicodeData.txt file in the repo I don't think we need to deal with third party node_modules either.